packages/react-components/react-overflow/stories/Overflow/Dividers.stories.tsx

```diff
@@ -18,22 +35,115 @@ export const Dividers = () => {
   return (
     <Overflow overflowDirection="start" padding={30}>
       <div className={styles.container}>
-        <TestOverflowItem id={'1'} groupId={'1'} />
-        <TestOverflowGroupDivider groupId={'1'} />
-        <TestOverflowItem id={'2'} groupId={'2'} />
-        <TestOverflowGroupDivider groupId={'2'} />
-        <TestOverflowItem id={'3'} groupId={'3'} />
-        <TestOverflowItem id={'4'} groupId={'3'} />
-        <TestOverflowGroupDivider groupId={'3'} />
-        <TestOverflowItem id={'5'} groupId={'4'} />
-        <TestOverflowItem id={'6'} groupId={'4'} />
-        <TestOverflowItem id={'7'} groupId={'4'} />
-        <TestOverflowGroupDivider groupId={'4'} />
-        <TestOverflowItem id={'8'} groupId={'5'} />
+        <OverflowItem id={'1'} groupId={'1'}>
+          <Button>Item 1</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'1'} />
+        <OverflowItem id={'2'} groupId={'2'}>
```

Comment:
Item 2 is overflowed with Item 1 but I wouldn't expect that based on how far I adjusted the Overflow width.


https://user-images.githubusercontent.com/93940821/210870406-2a642c7f-e397-41cf-9314-f227b96b3411.mp4

Reply:
Is that the case? because the overflow menu button itself also takes up space, so generally overflowing the first two items at once is quite normal